examples/client/client.c

```diff
@@ -1631,18 +1631,6 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
             err_sys("DisableExtendedMasterSecret failed");
         }
 #endif
-#if defined(HAVE_CURVE25519) && defined(HAVE_SUPPORTED_CURVES)
```

Comment:
Removing this code breaks the `./examples/client/client -t` use case to force use of X25519. Please repair this and make sure there is still a way to force X25519.